daml-lf/spec/daml-lf-1.rst

```diff
@@ -940,17 +994,17 @@ Then we define *well-formed expressions*. ::
     ——————————————————————————————————————————————————————————————— ExpGenMap (*)
       Γ  ⊢  〚e₁ ↦ e₁'; …; eₙ ↦ eₙ'〛: GenMap σ τ
 
-      τ contains no quantifiers
+      τ contains no quantifiers nor type synonyms
       ε  ⊢  τ : *     Γ  ⊢  e  : τ
```

Comment:
Similarly to `from_any` and `type_rep`, why not allow `τ ↠ τ'` and place the "no type synonym" requirements on `τ'` instead.